docs/Contributing/rituals/weekly-estimation.md

```diff
@@ -0,0 +1,38 @@
+## [Weekly Estimation](https://fleetdm.com/handbook/company/product-groups#sprint-ceremonies) 
+
+Purpose: Review user stories and bugs that have completed the drafting and specification process and add point estimates.
```

Comment:
Implicit: we know/agree on what a completely drafted, specced issue looks like (definition of ready).

Reply:
Good call out. We need a clear definition an examples of when a user story is ready to estimate.